en/docs/setup/changing-datasource-consent-management.md

```diff
@@ -0,0 +1,248 @@
+<td>The maximum number of milliseconds that the pool will wait (when there are no available 
+connections) for a connection to be returned before throwing an exception. You can enter 
+zero or a negative value to wait indefinitely.</td>
+</tr>
+<tr class="even">
+<td><strong>minIdle</strong></td>
+<td>The minimum number of active connections that can remain idle in the pool without 
+extra ones being created, or enter zero to create none.</td>
+</tr>
+<tr class="odd">
+<td><p><strong>testOnBorrow</strong></p></td>
+<td>Indicates whether objects will be validated before being borrowed from the pool. If 
+the object fails to validate, it will be dropped from the pool, and another attempt 
+will be made to borrow another.</td>
+</tr>
+<tr class="even">
+<td><p><strong>defaultAutoCommit</strong></p></td>
+<td>Indicates whether to commit database changes automatically or not.</td>
+</tr>
+<tr class="odd">
+<td><strong>validationInterval</strong></td>
+<td>The indication to avoid excess validation, and only run validation at the most, 
+at this frequency (time in milliseconds). If a connection is due for validation, but has 
+been validated previously within this interval, it will not be validated again.</td>
+</tr>
+<tr class="even">
+<td><strong>defaultAutoCommit</strong></td>
+<td><div class="content-wrapper">
+<p>This property is <strong>not</strong> applicable to the Carbon database in WSO2 
+products because auto committing is usually handled at the code level, i.e., the 
+default auto commit configuration specified for the RDBMS driver will be effective 
+instead of this property element. Typically, auto committing is enabled for RDBMS 
+drivers by default.</p>
+<p>When auto committing is enabled, each SQL statement will be committed to the 
+database as an individual transaction, as opposed to committing multiple statements 
+as a single transaction.</p>
+</td>
+</tr>
+</tbody>
+</table>
+    
+!!! info 
+    For more information on other parameters that can be defined in
+    the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat
+    JDBC Connection
+    Pool](http://tomcat.apache.org/tomcat-9.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
+
+!!! note "Sample Config"
+    If you have correctly configured, the `deployment.toml` file should have an entry 
+    similar to the following config. 
+    
+    **NOTE:** The following sample configuration is for a separate **PostgreSQL** consent 
+    management database.
+    
+    ```
+    [[datasource]]
+    id="WSO2CONSENT_DB"
+    url = "jdbc:postgresql://localhost:5432/gregdb"
+    username = "regadmin"
+    password = "regadmin"
+    driver = "org.postgresql.Driver"
+    jmx_enable=false
+    pool_options.maxActive = "80"
+    pool_options.maxWait = "60000"
+    pool_options.minIdle = "5"
+    pool_options.testOnBorrow = true
+    pool_options.validationQuery="SELECT 1"
+    pool_options.validationInterval="30000"
+    pool_options.defaultAutoCommit=false
```

Comment:
update changes related to [authentication.consent] data source here as well

Reply:
Fixed : 380b00c31c88d7a4de4a940ca60310a3367e5c37